core/server/common/pom.xml

```diff
@@ -54,6 +54,12 @@
       <groupId>com.google.guava</groupId>
       <artifactId>guava</artifactId>
     </dependency>
+    <!--aws sdk requires jaxb binding at runtime-->
+    <dependency>
+      <groupId>com.sun.xml.bind</groupId>
+      <artifactId>jaxb-impl</artifactId>
+      <scope>runtime</scope>
+    </dependency>
```

Comment:
I don't quite understand why this is required

Reply:
https://stackoverflow.com/questions/52502189/java-11-package-javax-xml-bind-does-not-exist
Without it, java 11 will throw a runtime error.